core/src/main/resources/hudson/model/AllView/noJob_fr.properties

```
@@ -21,3 +21,10 @@
 # THE SOFTWARE.
 
 Welcome\ to\ Jenkins\!=Bienvenue sur Jenkins !
+noJobDescription=Vos jobs Jenkins seront affich\u00e9s sur cette page. Pour commencer, vous pouvez mettre en place un build distribu\u00e9 ou commencer \u00e0 cr\u00e9er un projet.
+startBuilding=Commencer \u00e0 cr\u00e9er votre projet
+createJob=Cr\u00e9er un job
+setUpDistributedBuilds=Mettre en place un build distribu\u00e9
```

On pourrait peut être aligner tous les "setup" et réutiliser "configurer" comme pour `setUpCloud`?

```suggestion
setUpDistributedBuilds=Configurer un build distribu\u00e9
```